docs/api/cli-options.md

```diff
@@ -163,10 +250,13 @@ storybook doctor [options]
 
 Options include:
 
-| Option               | Description                                                                                                                                                                   |
-| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `-c`, `--config-dir` | Directory where to load Storybook configurations from<br/>`storybook doctor --config-dir .storybook`                                                                          |
-| `--package-manager`  | Sets the package manager to use when running the health check.<br/>Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook doctor --package-manager pnpm` |
+| Option                   | Description                                                                                                                                                                   |
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-c`, `--config-dir`     | Directory where to load Storybook configurations from<br/>`storybook doctor --config-dir .storybook`                                                                          |
+| `--package-manager`      | Sets the package manager to use when running the health check.<br/>Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook doctor --package-manager pnpm` |
+| `--debug`                | Outputs more logs in the CLI to assist debugging<br/>`storybook doctor --debug`                                                                                               |
+| `--disable-telemetry`    | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md)<br/>`storybook doctor --disable-telemetry`                                              |
+| `--enable-crash-reports` | Enable sending crash reprots to Storybook's telemetry. Learn more about it [here](../configure/telemetry.md).<br/>`storybook doctor --disable-telemetry`                      |
```

Comment:
When you're able, can you adjust as mentioned above in the `dev` command and provide the links to the appropriate sections